docs/instant.md

```diff
@@ -204,7 +204,7 @@ An example:
 
 ```js
 instant = Temporal.Instant.from('2019-03-30T00:45Z');
-new Date(instant.epochMilliseconds); // => 2019-03-30T00:45:00.000Z
+new Date(instant.epochMilliseconds).toISOString(); // => '2019-03-30T00:45:00.000Z'
```

Comment:
I'd remove the toISOString, the important thing is that you get a Date object at the correct instant.

Reply:
Ah, I'm comparing expressions using toString, and for Date that shows 'Fri Mar 29 2019 17:45:00 GMT-0700 (Pacific Daylight Time)'; the node repl representation shows the correct thing, however. I didn't know how to get that for automated testing, do you? Either way, I'll remove here and this can be one I audit manually in the future.

Reply:
Oh, this relates to what I commented above. I think what we want is the REPL representation. In Node you can get it with `util.inspect()`, I think.